app/src/main/kotlin/com/wire/android/ui/home/conversations/messages/ConversationMessagesViewModel.kt

```diff
@@ -74,6 +81,18 @@ class ConversationMessagesViewModel @Inject constructor(
     init {
         loadPaginatedMessages()
         loadLastMessageInstant()
+        observeNotifications()
+    }
+
+    private fun observeNotifications() = viewModelScope.launch {
+        observeNotifications.invoke().collect{
```

Comment:
@mchenani what do you think to have separate UseCase for listening single conversation notifications? Or maybe we should only listen for ping notifications?